src/utils/urlUtilities.ts

```diff
@@ -49,9 +48,8 @@ function getFragments(
  * from the outcome parameter. The IO APP will use the url and the outcome to give a feedback to the user
```

Comment:
consider to update the description (window.location.replace -> window.location.href)